src/main/java/net/wurstclient/hacks/AntiAfkHack.java

```diff
@@ -138,32 +151,28 @@ public void onUpdate()
 			if(processor.isDone())
 			{
 				PathProcessor.releaseControls();
-				timer = 40 + random.nextInt(21);
+				updateMovementValue();
 			}
 		}else
 		{
-			// set next block
+			
 			if(timer <= 0 || nextBlock == null)
 			{
 				nextBlock =
 					start.add(random.nextInt(3) - 1, 0, random.nextInt(3) - 1);
-				timer = 40 + random.nextInt(21);
+				updateMovementValue();
 			}
-			
 			// face block
 			WURST.getRotationFaker()
 				.faceVectorClientIgnorePitch(Vec3d.ofCenter(nextBlock));
-			
 			// walk
 			if(MC.player.squaredDistanceTo(Vec3d.ofCenter(nextBlock)) > 0.5)
 				MC.options.forwardKey.setPressed(true);
 			else
 				MC.options.forwardKey.setPressed(false);
-			
 			// swim up
 			MC.options.jumpKey.setPressed(MC.player.isTouchingWater());
 			
-			// update timer

```

Comment:
There are quite a few seemingly random formatting changes and removed comments here. Please revert.